release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -222,6 +222,15 @@ With this release, you can access the NMstate Operator and resources such as the
 
 IPsec is now supported on the IBM Cloud platform for clusters that use the OVN-Kubernetes network plugin, which is the default in {product-title} 4.14. For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-14-networking-ovn-kubernetes-ipsec-support-for-external-traffic"]
+==== OVN-Kubernetes network plugin support for IPsec encryption of external traffic
+
+{product-title} now supports encryption of external traffic, also known as north-south traffic. IPsec already supports encryption of network traffic between pods, known as east-west traffic. You can use both features in conjunction to provide full in-transit encryption for {product-title} clusters.
+
+To use this feature, you need to define an IPsec configuration tuned for your network infrastructure. For more information, refer to
+// TODO - enable
+// xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#nw-ovn-ipsec-north-south-enable_configuring-ipsec-ovn[Enabling IPsec encryption for external IPsec endpoints].
```

Comment:
I'm not sure that anchor will work `#nw-ovn-ipsec-enable_configuring-ipsec-ovn`. I'm guessing this is tied to 4.14 updates. If so, disregard my comment.

Reply:
@JoeAldinger, possible I'll have to fix it; I'll enable it once the content PR is merged. Thanks!